src/System.Private.CoreLib/shared/System/Runtime/Intrinsics/X86/Sse41.PlatformNotSupported.cs

```diff
@@ -11,9 +11,9 @@ namespace System.Runtime.Intrinsics.X86
     /// This class provides access to Intel SSE4.1 hardware instructions via intrinsics
     /// </summary>
     [CLSCompliant(false)]
-    public static class Sse41
+    public abstract class Sse41 : Ssse3
```

Comment:
12.12.2 Checking for SSE4.1 Support

Before an application attempts to use SSE4.1 instructions, the application should follow the steps illustrated in
Section 11.6.2, “Checking for SSE/SSE2 Support.” Next, use the additional step provided below:

Check that the processor supports SSE4.1 (if CPUID.01H:ECX.SSE4_1[bit 19] = 1), SSE3 (if
CPUID.01H:ECX.SSE3[bit 0] = 1), and SSSE3 (if CPUID.01H:ECX.SSSE3[bit 9] = 1).